samples/snippets/speech_to_storage_beta_draft.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START speech_transcribe_with_speech_to_storage_beta]
+
+from google.cloud import speech_v1p1beta1 as speech
+from google.cloud import storage
+import json
+
+
+def export_transcript_to_storage_beta(
+    input_storage_uri,
+    output_storage_uri,
+    encoding,
+    sample_rate_hertz,
+    language_code,
+    bucket_name,
+    bucket_prefix,
```

Comment:
Is it common convention to call this bucket_prefix, can it be named object_name?

Or perhaps function can take gcs uri like gs://bucket/location/file path?

Reply:
Renamed to ```object_name```.